terraform/instances.tf

```diff
@@ -403,11 +449,14 @@ resource "aws_instance" "scheduler" {
 
   provisioner "remote-exec" {
     inline = [
+      "chmod +x /tmp/install_base_packages.sh",
+      "sudo /tmp/install_base_packages.sh core/builder-scheduler",
+
       "sudo mv /home/ubuntu/hab-sup.service /etc/systemd/system/hab-sup.service",
       "sudo systemctl daemon-reload",
       "sudo systemctl start hab-sup",
       "sudo systemctl enable hab-sup",
-      "sudo hab svc load core/builder-scheduler --group ${var.env} --bind router:builder-router.${var.env} --bind datastore:builder-datastore.${var.env} --bind depot:builder-api.${var.env} --strategy at-once --url ${var.depot_url} --channel ${var.release_channel}"
```

Comment:
Whoa, thanks for catching that. That change shouldn't have been there in the first place.